adcm_aio_client/core/objects/_imports.py

```diff
@@ -1 +1,33 @@
-class ClusterImports: ...
+from typing import TYPE_CHECKING, Collection, Self, Union
+
+from adcm_aio_client.core.types import Endpoint, Requester
+
+if TYPE_CHECKING:
+    from adcm_aio_client.core.objects.cm import Cluster, Service
+
+
+class Imports:
+    def __init__(self: Self, requester: Requester, path: Endpoint, imports_data: dict) -> None:
```

Comment:
you can just take parent and use it's `get_own_path` for detecting path to imports endpoint

that way your API will be cleaner